prihlas-me.php

```diff
@@ -0,0 +1,23 @@
+<?php
+session_start();
```

Comment:
Možná to budeš mít dál, ale tohle bych pak asi přesunul do `sdilene.php` ať se to startuje všude, ne?

Reply:
to bude opravdu as nekde dal (mimo ukol)